src/applications/avs/tests/components/YourAppointment.unit.spec.jsx

```diff
@@ -36,7 +40,7 @@ describe('Avs: Your Appointment', () => {
     const avs = replacementFunctions.cloneDeep(avsData);
     delete avs.reasonForVisit;
     delete avs.diagnoses;
-    delete avs.vitals;
+    avs.vitals = [];

```

Comment:
I made this change to not have to add more guards in the render component. I believe it's safe to make because the vets-api AVS model will always send the vitals as an empty array even if nothing exists in the source.